subsys/metrics/src/main/java/org/commonjava/indy/IndyMetricsManager.java

```diff
@@ -85,6 +108,65 @@ public void initMetric()
             logger.error( e.getMessage() );
             throw new RuntimeException( e );
         }
+
+        // Set up TransportMetricConfig
+        if ( config.isMeasureTransport() )
+        {
+            final String measureRepos = config.getMeasureTransportRepos();
+            final List<String> list = new ArrayList<>();
+            if ( isNotBlank( measureRepos ) )
+            {
+                String[] toks = measureRepos.split( "," );
+                for ( String s : toks )
+                {
+                    s = s.trim();
+                    if ( isNotBlank( s ) )
+                    {
+                        if ( s.indexOf( ":" ) < 0 )
+                        {
+                            s = MAVEN_PKG_KEY + ":" + remote.singularEndpointName() + ":" + s; // use default
+                        }
+                        list.add( s );
+                    }
+                }
+            }
+            transportMetricConfig = new TransportMetricConfig()
```

Comment:
Could we just make the IndyMetricsConfig implement TransportMetricConfig? Or, maybe just an inner class of IndyMetricsConfig?

Reply:
I will make IndyMetricsConfig implement TransportMetricConfig.